wherehows-web/app/components/dataset-compliance.ts

```diff
@@ -1173,7 +1173,16 @@ export default class DatasetCompliance extends Component {
         });
       }
 
-      const policy = Object.assign({}, getProperties(currentPolicy, ['datasetClassification', 'complianceEntities']));
+      const policy = Object.assign(
```

Comment:
Confused why you need to `Object.assign` an empty object here. Doesn't `getProperties` create you a new object with those keys anyway?